docs/UserGuide.md

```diff
@@ -42,124 +42,125 @@ CCACommander Ultra Promax Xtra 9000PLUS is the one-stop app for CCA Heads to man
+### View Members of Event : `viewEvent`
 
 Lists all the members of a specified event index.
 
-Format: `view /event EVENT_INDEX`
+Format: `viewEvent EVENT_INDEX`
 * Views the members of the event at the specified `EVENT_INDEX`.
 * The index refers to the index number shown in the displayed event list.
 * The index **must be a positive integer** that is within the range of the length of the event list.
 
 Examples:
-* `view /event 1` displays members of the 1st event in the event list.
-* `view /event 10` displays members of the 10th event in the event list.
+* `viewEvent 1` displays members of the 1st event in the event list.
+* `viewEvent 10` displays members of the 10th event in the event list.
 
-### View Events of Member : `view /member`
+### View Events of Member : `viewMember`
 
 Lists all the events of a specified member index.
 
-Format: `view /member MEMBER_INDEX`
+Format: `viewMember MEMBER_INDEX`
 
 * Views the events of the member at the specified `MEMBER_INDEX`.
 * The index refers to the index number shown in the displayed member list.
 * The index **must be a positive integer** that is within the range of the length of the member list.
 
 Examples:
-* `view /member 1` displays members of the 1st member in the member list.
-* `view /member 10` displays members of the 10th member in the member list.
+* `viewMember 1` displays members of the 1st member in the member list.
+* `viewMember 10` displays members of the 10th member in the member list.
 
-### Create an Event : `create /event`
+### Create an Event : `createEvent`
 
 Creates a new event and adds it to the database.
 
-Format: `create /event n/EVENT_NAME [l/LOCATION] [d/DATE]`
+Format: `createEvent n/EVENT_NAME [l/LOCATION] [d/DATE]`
 
 Examples:
-* `create /event n/Party l/Raffles Hall d/16-09-2023` creates an event `Party` in CCACommander.
-* `create /event n/Combined Hall Ensemble Concert d/16-02-2024` creates an event `Combined Hall Ensemble Concert` in CCACommander.
+* `createEvent n/Party l/Raffles Hall d/16-09-2023` creates an event `Party` in CCACommander.
+* `createEvent n/Combined Hall Ensemble Concert d/16-02-2024` creates an event `Combined Hall Ensemble Concert` in CCACommander.
 
-### Delete an Event: `delete /event`
+### Delete an Event: `deleteEvent`
 
 Deletes the event at the specified index.
 
-Format: `delete /event EVENT_INDEX`
+Format: `deleteEvent EVENT_INDEX`
 
 * Deletes the event at the specified `EVENT_INDEX`.
 * The index refers to the index number shown in the displayed event list.
 * The index **must be a positive integer** that is within the range of the length of the event list.
 
 Examples:
-* `delete /event 1` deletes the 1st event in the event list.
-* `delete /event 10` deletes the 10th event in the event list.
+* `deleteEvent 1` deletes the 1st event in the event list.
+* `deleteEvent 10` deletes the 10th event in the event list.
 
-### Add Member to an Event: `add /member /event`
+### Add Member to an Event: `enrol`
```

Comment:
Do we want to change the heading of the enrol function to be something like Link a Member to an Event? No strong preference here

Reply:
I feel that link and unlink might be clearer in communicating what exactly the command does to the user.

Reply:
Ok will introduce the change.